model_to_project_task/action.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+# © 2015 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api, _
```

Comment:
Thanks to notice, I'll fix but I never seen pep or oca guidelines on this subject.
Do you have seen something ?